### PR TITLE
Xchourio/viirs djibouti ethiopia

### PR DESCRIPTION
**Pull request by xchourio**
Test in:
http://shortfin01.iri.columbia.edu:9999/fbfmaproom/ethiopia-ond
http://shortfin01.iri.columbia.edu:9999/fbfmaproom/ethiopia

@aaron-kaplan  before sharing the link on Trello, I want to know if those adjustments are correct. Thanks!

### CODE REVIEW

**Comment by aaron-kaplan** [comment]
According to Max in [Trello](https://trello.com/c/Eo1UDPHi/26-ethiopia-djibouti-viirs-vegetation-index):

> we’ll want it for the following months: June, July, August (for OND version of map tool) and November, December, January (for MAM version of map tool). don’t need the seasonal averages

So please remove the following:
ndvi-viirs-mam
ndvi-viirs-feb-mam
ndvi-viirs-ond
ndvi-viirs-prev-seas-ond
ndvi-viirs-prev-seas-mam

Also, please change the labels for the old NDVI column to "NDVI (MODIS 6)", as we did for Djibouti.

Other than that, it looks good as far as I can see, but please confirm the Djibouti changes by downloading the new version to fbf-candidate, opening it in the application, and comparing the numbers for those columns of the table to what we have in production. If there are differences, something is wrong.

**Comment by aaron-kaplan** [comment]
Go ahead and share with FIST.

**Comment by aaron-kaplan** [comment]
@xchourio I just opened /data/aaron/fbf-candidate/djibouti/ndvi-viirs-jas.zarr and it still has the bounding box of Ethiopia. Before we merge this, please re-download the Djibouti VIIRS datasets and confirm that the numbers in the table don't change, by comparing the table on the internal test server to the production one.

File: fbfmaproom/fbfmaproom-sample.yaml [hunk 1]
```diff
@@ -2062,17 +2134,6 @@ countries:
...
-                    label: NDVI (VIIRS)
-                    path: djibouti/ndvi-viirs-mam.zarr
-                    var_names:
-                        value: NDVI
-                        lat: Y
-                        lon: X
-                        time: T
-                    colormap: precip
-                    lower_is_worse: yes
-                    format: number4
```
Comment (aaron-kaplan) [inline fbfmaproom/fbfmaproom-sample.yaml:2075]: Why did you delete this?

Comment (xchourio) [reply fbfmaproom/fbfmaproom-sample.yaml]: Thanks! actually I don't know why I did it, I guess I got confused with the name and didn't realize it was Djibouti......fixed!

**Comment by xchourio** [comment]
> Djibouti VIIRS spatial domain change has been validated and data files copied to prod.
> 
> We need to add VIIRS to southern-oromia and southern-oromia-ond. They use the same data files as ethiopia and ethiopia-ond.

@aaron-kaplan  ; Before continuing ..... Same VIIRS as Ethiopia (Nov, Dec, Jan ) ? (see the last commit)

**Comment by aaron-kaplan** [comment]
Yes, same months as for ethiopia. I see you did it for MAM, we also need it for OND.

**Review by aaron-kaplan — APPROVED** [review]
